pkg/daemon/ceph/cleanup/radosnamespace.go

```diff
@@ -91,3 +102,34 @@ func BlockPoolCleanup(context *clusterd.Context, clusterInfo *cephclient.Cluster
 	logger.Infof("successfully cleaned up CephBlockPool %q resource", poolName)
 	return nil
 }
+
+func AddClientIPsToBlocklist(context *clusterd.Context, clusterInfo *cephclient.ClusterInfo, images []cephclient.CephBlockImage, poolName, radosNamespace string) error {
```

Comment:
`AddClientIPsToBlocklist` to `addClientIPsToBlocklist`

Reply:
had changed it to `blocklistClientIPs` before reading this comment!